edgedetector/solver/evaluator/cost/curvature/curvature_cost_evaluator.py

```diff
@@ -1,9 +1,23 @@
 """
 This class implements evaluation of genotype's curvature cost function.
 """
-from edgedetector.solver.evaluator.evaluator import Evaluator
+from edgedetector.solver.evaluator.cost.cost_evaluator import CostEvaluator
+
+
+class CurvatureCostEvaluator(CostEvaluator):
+
+    def evaluate_window(self, row, column):
+        if not self.matrix[row, column]:
+            return 0
+        neighbours = self._find_neighbour_edge_pixels(row, column)
+        if len(neighbours) >= 3:
```

Comment:
When there are 3 or more neighbours it is 100% that there exist pair of neighbour pixels that (with center pixel) create line that turns by more than 45%